packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -362,6 +382,14 @@ export class Viewer {
               'enable-pointer-events ': window.PointerEvent != null,
             })}
           >
+            {this.getStreamAttributes().depthBuffers?.enabled && (
+              <canvas
+                ref={(ref) => (this.depthBufferCanvasElement = ref)}
+                class="depth-buffer-canvas"
+                width={1}
+                height={1}
+              ></canvas>
+            )}
```

Comment:
This isn't really required either :) You should be able to create a new canvas element and draw into it without it actually adding it to the DOM.